jest.config.js

```diff
@@ -1,14 +1,8 @@
 module.exports = {
     //globalSetup: "./test-environment/setup.js",
     //globalTeardown: "./test-environment/teardown.js",
-    //testEnvironment: "./test-environment/environment.js",
+    testEnvironment: "./test-environment/environment.js",
     preset: "jest-playwright-preset",
-    testEnvironmentOptions: {
```

Comment:
where browsers that will run is set,  when this is removed?

Reply:
jest-playwright.config.js